src/features/DebugSession.ts

```diff
@@ -617,25 +630,24 @@ export class PickPSHostProcessFeature extends LanguageClientConsumer {
         }
     }
 
-    private async pickPSHostProcess(): Promise<string | undefined> {
+    private async pickPSHostProcess(): Promise<number | undefined> {
```

Comment:
@JustinGrote I went through and plumbed these _all_ to be `number`.

Reply:
Looks like build didn't trigger on force push for some demonic reason. Lets try your re-push